tiledb/ml/models/sklearn.py

```diff
@@ -23,6 +25,15 @@ class SklearnTileDBModel(TileDBModel[BaseEstimator]):
     Framework = "SKLEARN"
     FrameworkVersion = sklearn.__version__
 
+    def __init__(
+        self,
+        uri: str,
+        namespace: Optional[str] = None,
+        ctx: Optional[tiledb.Ctx] = None,
+        model: Optional[Model] = None,
+    ):
+        super().__init__(uri, namespace, ctx, model)
```

Comment:
Redundant override

Reply:
The override is used to keep the API `model` argument even though in the underlying base class we refer it as `artifact`